code/modules/mob/living/living.dm

```diff
@@ -1845,3 +1845,37 @@ default behaviour is:
 			return TRUE
 	return FALSE
 
+/mob/living/proc/flee(atom/target, upset = FALSE)
+	var/static/datum/automove_metadata/_flee_automove_metadata = new(
+		_move_delay = null,
+		_acceptable_distance = 7,
```

Comment:
Aren't these values backwards? 7 tiles distance for calm but 2 for annoyed

Reply:
flee -> upset, annoyed -> not upset